bob-backend/src/pearl/holder.rs

```diff
@@ -158,14 +158,25 @@ impl Holder {
         }
     }
 
+    // NOTE: stack + heap size (in fact that's serialized size)
+    // NOTE: can be calculated like `Data::from(data).len()`, but that's less efficient
+    fn calc_data_size(data: &BobData) -> u64 {
+        (std::mem::size_of::<BobData>() + data.inner().len()) as u64
```

Comment:
Think of [`size_of_val()`](https://doc.rust-lang.org/std/mem/fn.size_of_val.html#)